test/guest/cockpit-fedora-22.setup

```diff
@@ -26,6 +29,37 @@ device-mapper-multipath \
 atomic \
 "
 
+COCKPIT_BUILD_DEPS="\
```

Comment:
We would use `mock --installdeps`.  We can do that, and I was on the edge.  Hrd coding the packages seemed simpler to get started, but there shouldn't be any downsides to taking the builddeps from a srpm.